syft/pkg/cataloger/golang/binary_cataloger.go

```diff
@@ -16,15 +16,6 @@ import (
 
 const catalogerName = "go-module-binary-cataloger"
 
-// current mime types to search by to discover go binaries
```

Comment:
Nice glad these are going to their own thing. Syft having mime-type destructed to its own thing is a great level up